src/components/License.tsx

```diff
@@ -28,6 +33,7 @@ const LicenseCard: FC<LicenseProps> = (props) => {
             <Link to={props.license.features?.dlr_license_url_no ? props.license.features.dlr_license_url_no : ''}>
               <img src={props.license.features?.dlr_license_url_image} alt={props.license.features?.dlr_license_code} />
             </Link>
+            <br />
```

Comment:
br er foretrukket brukt samantisk fremfor for  plassering. Man kan heller pakke inn elementet eller stile det med margin 
fra MDN web docs : "Note: Do not use <br> to create margins between paragraphs; wrap them in <p> elements and use the CSS margin property to control their size."

Reply:
Enig, sleit bare med å få css til å gjøre jobben. Wrappet inn i en div istedet.